cmd/aries-js-worker/src/aries.js

```diff
@@ -176,14 +176,40 @@ const Aries = function(opts) {
             return () => {quit = true}
         },
 
+        /**
+         * DIDExchange methods - Refer to [OpenAPI spec](docs/rest/openapi_spec.md#generate-openapi-spec) for
+         * input params and output return json values.
+         */
         didexchange: {
             pkgname: "didexchange",
+
+            /**
+             * Creates a DID Exchange Invitation.
+             *
+             * @param text - json document
```

Comment:
`text` is an odd argument name. should we fix?

Reply:
[Updated](https://github.com/hyperledger/aries-framework-go/compare/e1b402f9989822b55eebe082c9a4acf49252fba1..e02e4a41644e17e908307b9dbfbe7b1710b94960#diff-a43de5ddb6dcf66800367a9232d43f90R189) to name of the existing param to req.